docs/faq.md

```diff
@@ -29,6 +29,44 @@ Rails.application.routes.draw do
 end
 ~~~
 
+### Another method for using this gem alongside standard Devise (updated May 2018)
+
+Some users have been experiencing issues with using this gem alongside standard Devise, with the `config.enable_standard_devise_support = true` method.
+
+Another method suggested by [jotolo](https://github.com/jotolois) is to have separate child `application_controller.rb` files that use either DeviseTokenAuth or standard Devise, which all inherit from a base `application_controller.rb` file. For example, you could have an `api/v1/application_controller.rb` file for the API of your app (which would use Devise Token Auth), and a `admin/application_controller.rb` file for the full stack part of your app (using standard Devise). The idea is to redirect each flow in your application to the appropriate child `application_controller.rb` file. Example code below:
```

Comment:
```https://github.com/jotolo``` is the right address ;) Thank you the mention

Reply:
thanks for catching that!